owslib/util.py

```diff
@@ -17,6 +18,7 @@
 from owslib.etree import etree, ParseError
 from owslib.namespaces import Namespaces
 from six.moves.urllib.parse import urlsplit, urlencode, urlparse, parse_qs, urlunparse
+import traceback
```

Comment:
Why are we adding `traceback` here?

Reply:
Not sure how that got added. I'll remove it.